aruba/features/renew/renew-with-csr-service.feature

```diff
@@ -30,29 +33,39 @@ Feature: renew action with `-csr service` option
       And it should write private key to the file "k.pem"
       And it should write certificate to the file "c.pem"
       And it should output Pickup ID
-    When I renew the certificate in TPP using the same Pickup ID with flags -no-prompt -cert-file c1.pem -key-file k1.pem -csr service -key-password newPassw0rd!
+    When I renew the certificate in TPP using the same Pickup ID with flags -no-prompt -cert-file c1.pem -key-file k1.pem -csr service -key-password Passcode123!
     Then it should fail with "Status: 400"
 
   Scenario: renew service-generated-CSR certificate in TPP with `-csr service` option
-    Given I enroll random certificate using TPP with -csr service -no-prompt -key-file k.pem -cert-file c.pem -key-password newPassw0rd!
+    Given I enroll random certificate using TPP with -csr service -key-file k.pem -cert-file c.pem --key-password Passcode123!
```

Comment:
Because that flag was not taking effect. I added a truth table analyzing those cases when creating a temporary password for both `enroll` and `renew` actions.